### ISSUE TITLE
im new to spfx, im facing some problem in pnp people picker(ProcessHttpClientResponseException: Error making HttpClient request in queryable: [400] at new ProcessHttpClientResponseException

### ISSUE THREAD

**Issue by abekcm20** [pnp/pnpjs#2045]
<PeoplePicker
                      context={this.props.context}
                      titleText=""
                      personSelectionLimit={3}
                      groupName={""}
                      showtooltip={true}
                      //required={true}
                      disabled={false}
                      //onChange={this._getPeoplePickerItems}
                      showHiddenInUI={false}
                      principalTypes={[PrincipalType.User]}
                      resolveDelay={1000}
                      selectedItems={this.getTrainer}
                      //onChange={this.getTrainer}
                      ></PeoplePicker>
                      </div>

public getTrainer(items:any[]){
    let ppl:any[]=[];
    items.map((item)=>{
      ppl.push(item.id);
    });
    this.setState({Trainer:ppl});
  }

pnp.sp.web.lists.getByTitle('Students').items.add({
        StudentName:this.state.StudentName,
        Email:this.state.Email,
        Mobile:this.state.Mobile,
        Address:this.state.Address,
        Referral:this.state.Referral,
        JoinImmediately:this.state.JoinImmediately,
        Date:this.state.Date,
        Year:this.state.Year,
        Skills:this.state.Skills,
        TrainerId:{results:this.state.Trainer},
        Department: {
          '__metadata': { "type": "SP.Taxonomy.TaxonomyFieldValue" },
          'Label': this.state.Department[0].name,
          'TermGuid': this.state.Department[0].key,
          'WssId': '-1'
        }
      }).then(response =>{
        alert('Adding an item has been completd Successfully');
      });

**Comment by abekcm20** [comment]
allow multiple selections in SharePoint list
